include/openPMD/IO/AbstractIOHandlerHelper.hpp

```diff
@@ -60,8 +60,8 @@ createIOHandler(
  */
 std::shared_ptr< AbstractIOHandler >
 createIOHandler(
-    std::string path,
-    AccessType accessType,
-    Format format,
-    std::string const & options = "{}" );
+        std::string path,
+        Access accessType,
```

Comment:
You probably meant to rename this one, too? `Access access`

Reply:
they were not user-facing so I did not bother too much :) but can do.

Reply:
The parameter name in the parallel overload of this is changed, so it's probably good to keep it consistent ;)

Reply:
Serious though: shall I rename them or will this lead to more conflicts with your streaming branch?

Reply:
Are you referring to `accessTypeBackend` and `…Frontend`? Think it's no problem if you rename those, there are no streaming-specific usages in the ADIOS2 backend.